### ISSUE TITLE
Add a way to save/persist max health

### ISSUE THREAD

**Issue by RoboMWM** [Dankrushen/SetMyHealth#1]
There was one other plugin that attempted to do this but it fails to do so: http://dev.bukkit.org/bukkit-plugins/morehearts/ because of an NPE. Seems doable otherwise?

**Comment by Dankrushen** [comment]
So basically save the amount of hearts, and the amount of health on quit, then give it all back on the next join?

**Comment by RoboMWM** [comment]
Yes, although https://www.spigotmc.org/resources/customhealth.8235/ has been updated to support this. However, it doesn't work when MassiveCore (library for Factions) is installed. I'm trying to see if this is fixable by adding soft dependencies.

**Comment by RoboMWM** [comment]
So I took a look at CustomHealth and it isn't saving or restoring hearts, just setting maxhealth. So it seems this is not needed, as long as you set maxhealth on respawn (and join?) to maintain the maxhealth setting.

**Comment by Dankrushen** [comment]
Sorry for not doing anything, I've been busy, and I've sort of abandoned working on Minecraft plugins, I still do them, it's just not my main focus.

**Comment by RoboMWM** [comment]
That's fine, I've completely ridded myself of MassiveJunk, and am using a different plugin (one that I originally wanted to use before I discovered MassiveCore screwing around with maxHealth) to accomplish what I want. Thanks for responding though!
